UPGRADE-4.0.md

```diff
@@ -0,0 +1,9 @@
+UPGRADE FROM 3.x to 4.0
+=======================
+
+### DoctrineBridge and Validator
+
+ * If you were using the `StaticMethodLoader` to create `UniqueEntity`
```

Comment:
It would be, however the current behavior of `StaticMethodLoader` is to do the bare minimum and to let the implementation of the loader method do all the heavy lifting work.

That's why I didn't implement the support for TA constraints in there. But, yeah, it could be done, I guess... I will add a note in the other PR.